frontend/functions/delete-contact-person.ts

```diff
@@ -0,0 +1,15 @@
+import { ContactPerson } from "@frontend/components/patient/types/contact-person";
+import { serverURL } from "@frontend/config";
+
+export default async function deleteContactPerson(person: ContactPerson, id: number) {
```

Comment:
Is there a reason why this function has its own file? If it's not being reused then wouldn't it better to keep the function in the same file where it's called?

Reply:
its just for cleanliness, because the location of this function's call is already like 20 lines long